pkg/common/telemetry/inmem_test.go

```diff
@@ -11,8 +11,6 @@ import (
 )
 
 func TestInMem(t *testing.T) {
-	enabled := true
-	disabled := false
 
```

Comment:
Remove this empty line at the beginning of the TestInMem function to satisfy the linter.

Reply:
Thanks! fixed it.